src/resources/MachineLearning/ModelInformation/ModelInformationInterfaces.ts

```diff
@@ -209,9 +209,71 @@ export interface BuildingStatsSS {
     headerCount: number;
     meanSnippetLength: number;
 }
+
+/**
+ * Snippet per doc details
+ */
+export interface SnippetsPerDocument {
+    min: number;
```

Comment:
Why not add comments for these?

Reply:
because I added a global comment for this stats above, but I could also add for each :)